### ISSUE TITLE
Cannot use CLI tool

### ISSUE THREAD

**Issue by Fluffy-Bean** [timmo001/system-bridge#2898]
### Description

I've used the guide on the website to install the project through pip on Arch, but I keep getting this error whenever I try to use the CLI tool.

### What Platform / OS are you running?

Linux

### What version are you running?

N/A

### Anything in the logs or a references that might be useful?

```txt
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/home/fluffy/.local/lib/python3.11/site-packages/systembridgecli/__main__.py", line 11, in <module>
    from systembridgeshared.const import SECRET_API_KEY, SETTING_PORT_API
ImportError: cannot import name 'SECRET_API_KEY' from 'systembridgeshared.const' (/home/fluffy/.local/lib/python3.11/site-packages/systembridgeshared/const.py)
```
```


### Additional information

As I cannot access the CLI tool, I'm not too sure what version I'm running

**Comment by timmo001** [comment]
#2888

**Comment by timmo001** [comment]
This should now be fixed in 4.0.0
